shell/edit/provisioning.cattle.io.cluster/index.vue

```diff
@@ -449,10 +450,14 @@ export default {
       }
     },
 
+    filteredSubTypes() {
```

Comment:
Moving the filtering to this step (so that 'hidden' provisioners are still in the `subtypes` array) ensures that if a user creates a cluster using one of these provisioners then disables the associated driver, the right ui still loads when editing that cluster. Not sure if it really makes sense for a user to get themselves into that scenario, but it feels like we should load the right UI if they do anyway